packages/compass-components/src/hooks/use-focus-ring.ts

```diff
@@ -16,7 +16,7 @@ export const focusRingStyles = css({
     bottom: 3,
     left: 3,
     borderRadius: spacing[1],
-    boxShadow: `0 0 0 0 ${uiColors.focus}`,
+    boxShadow: '0 0 0 0 transparent',
```

Comment:
Why are we removing the focus outline color?

Reply:
Moved to the `option-editor` so we aren't impacting everything. We want the hover color to not transition from the focus color, setting it to transparent makes the hover not flash the focus color.